INFO     [17:49:04] __main__.foo(): foo
ERROR    [17:49:04] __main__.foo(): error: code = 5
